### ISSUE TITLE
Feature request: factors in col_types

### ISSUE THREAD

**Issue by qdread** [tidyverse/readxl#681]
I would like to request that the `col_types` argument of `read_excel()` and related functions allow specifying factor as a column type. This is especially relevant for columns that are automatically guessed to be numeric but that I would like to treat as factors (e.g., numerical IDs of experimental blocks). Of course it is possible to coerce those columns to factor in a later step, but it would be helpful if it were possible to do this within `read_excel()`.

**Comment by jennybc** [comment]
Yes. readxl is going to get a major upgrade re: column type specification and that will absolutely include supporting factors.

**Comment by Sibojang9** [comment]
Any update on this request? has this been implemented. Thanks.
